test-case/check-kmod-load-unload.sh

```diff
@@ -50,6 +50,23 @@ do
     ## - 1: remove module section
     func_lib_setup_kernel_last_line
 
+    # 1. check dsp status before removal, only remove module when dsp is suspended.
+    # 2. it will take some time (about 10s) for dsp to become suspended from active
+    # after module insertion on some platforms, here we check dsp status, and only continue
+    # next removal when dsp is suspended.
+    dlogi "wait dsp power status to become suspended"
+    declare I=0 # used to check if for loop is early exit
+    for i in $(seq 1 15)
+    do
+        [[ $(sof-dump-status.py --dsp_status $SOFCARD) == "suspended" ]] && break
```

Comment:
I find come problem, here is `lspci` order  
so don't need to use `SOFCARD`

so need to confirm `lspci` refer audio order
here you can direct use 0 instead of `SOFCARD` add the comment at here

Reply:
done @Bin-QA